omnia.yml

```diff
@@ -50,6 +50,12 @@
 - name: Import playbook to install ROCm on nodes
   ansible.builtin.import_playbook: utils/rocm_installation.yml
 
+# By default, this will run verify steps for Intel Gaudi installation.
+# If you want to skip this verify, you can add `--skip-tags verify_intel_gaudi_install` option to ansible-playbook.
+- name: Import playbook to verify Intel Gaudi Installation
+  ansible.builtin.import_playbook: utils/verify_intel_gaudi_installation.yml
```

Comment:
We can remove this invocation from omnia.yml. We will discuss and add back if required later

Reply:
We have removed the invocation from omnia.yml. and added TODOs in the code so we keep track of this. As discussed before It's important that the verification is invoked during the default provisioning path.